### PR TITLE
Development

### PR DESCRIPTION
**Pull request by jsariasgeek**
The project is ready according to the instructions.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -73,4 +77,143 @@ header div {
...
 .img-nav{
     max-height: 30px;
+
+
+}
+
+.like-share{
+   display: flex; 
+   justify-content: flex-end;
+   /* flex-direction: column; */
```
Comment (zlayabekrija) [inline css/main.css]: please remove commented out properties

File: index.html [hunk 2]
```diff
@@ -28,7 +29,186 @@
         </nav>
     </header>
     <main>
+        <br>
+            <div class="main">
+                    <div class="video_coments">
+                        <div class="izquierda">
+                           <div class="video_container">
+                               <iframe height="500px" src="https://www.youtube.com/embed/q-U72ky8lFs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (zlayabekrija) [inline index.html]: should be only digit no px,em,% etc.

File: index.html [hunk 3]
```diff
@@ -28,7 +29,186 @@
...
+                                <h2>El pastor</h2>
+                                <p>Ma'am. How far does the 8:00 train go? San Francisco</p>
+                            </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
     </main>
-    <footer></footer>
+    <footer>
```
Comment (zlayabekrija) [inline index.html]: footer is empty please remove it.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Everything looks perfect except of navbar, please adjust it, to look more youtube like.
You can use fontawsome instead of images.
Please follow up with notes in the code.
On line 15 of your css file `vertical-align: middle;` doesn't have any function, please remove it.
Please make requested changes,and submit a new review request.
If you have any questions feel free to contact me on slack.
Happy coding.

**Comment by jsariasgeek** [comment]
Ok I have applied your suggestions. Please review again and let me know.
Thanks friend.

File: index.html (hunk 3 above)
Comment (edlingao) [reply index.html]: Fixed footer and comments on css

**Review by zlayabekrija — APPROVED** [review]
As we discussed previously, nicely done from your side, thumbs up 👍  from me, and good luck with next projects.
Happy coding 😄

**Pull request merged**